de.metas.business/src/main/java/de/metas/customs/CustomsInvoiceRepository.java

```diff
@@ -382,4 +385,18 @@ private static CustomsInvoiceLineId extractCustomsInvoiceLineId(final I_M_InOutL
 		return CustomsInvoiceLineId.ofRepoId(record.getC_Customs_Invoice_ID(), record.getC_Customs_Invoice_Line_ID());
 	}
 
+	public void setExportedInCustomsInvoice(final InOutId shipmentId)
```

Comment:
A repository should be about loading and storing entities. Thi that regard, this repo is already pretty much f'ed up, but we should not make things worse. Therefore, i would say please move this method to `IInoutDAO`

Reply:
done